### ISSUE TITLE
counters not appearing

### ISSUE THREAD

**Issue by dpfahlbusch** [FrDH/mmenu-js#1017]
Hey,
added the counters like in the docs, but they dont appear.

Maybe something changed or the docs are not up to date.

i used:
```javascript
document.addEventListener(
      "DOMContentLoaded", () => {
        new Mmenu( "#my-menu", {
           "counters": true
        });
      }
   );
```

**Comment by FrDH** [comment]
HI,
Thank you for bringing this to my attention. I'll have to investigate why the counters no longer work.

**Comment by FrDH** [comment]
Should be fixed in v8.0.8

**Comment by dpfahlbusch** [comment]
Works again, thank you. 👍
